www/configurator/src/app/css-transpiler/css-transpiler.worker.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+
+/**
+  * @fileoverview Web worker for the CssTranspiler Class, to handle css transpilation.
+	* 		Simply responds to messages containing the template CSS, and css vars, and responds with the newly transpiled css.
+  */
+
+const postcss = require('postcss');
+const customProperties = require('postcss-custom-properties');
+
+onmessage = function(event) {
+
+	// Obtain the templateCss and cssvars from event
+	const templateCss = event.data.templateCss;
```

Comment:
anything you can think of - obvious ones are missing data - invalid vars might be a good one - we might need to let the users know what went wrong